server/src/weaverbird/backends/pypika_translator/operators.py

```diff
@@ -5,6 +5,11 @@ class RegexOp(Enum):
     REGEXP = auto()
     SIMILAR_TO = auto()
     CONTAINS = auto()
+    REGEXP_LIKE = auto()
+    REGEXP_SUBSTR = auto()
+    REGEXP_EXTRACT = auto()
+    REGEXP_MATCH = auto()
+    REGEXP_CONTAINS = auto()
```

Comment:
These cases are not tested yet ?
Shouldn't we add the corresponding unit test ?

Reply:
Only `REGEXP_LIKE` is tested in this Pull Request, is it better for to remove the others ? 🤔 
Because i added them after checking some regex SQL syntax available !

Reply:
I mean making something similar to this :
https://github.com/ToucanToco/weaverbird/blob/master/server/tests/backends/sql_translator_unit_tests/test_filter_translators.py#L124

Reply:
Alright, i see, you mean something like this : https://github.com/ToucanToco/weaverbird/pull/1418/commits/e8c542b9f513e4f415570dff7b89ad129cbc78a5 ?